pyproject.toml

```diff
@@ -54,3 +54,23 @@ changelog = "https://github.com/jazzband/django-constance/releases/"
 
 [tool.setuptools.packages.find]
 include = ["constance*"]
+
+
+[tool.ruff]
+# Same as Black.
+line-length = 88
```

Comment:
```suggestion
line-length = 99
```

Reply:
In black it's 88. Also, 120 is commonly used. 99 is something not usual. I prefer to leave 88 or change to 120

Reply:
AFAIK the PEP8 never talks about 88 or 120, I was suggesting 99 because that's the maximum the PEP8 talks about. I'm not against 120 either but it seems pretty uncommon to me, should it be 119 to account for the `\n`?

Reply:
The goal is mostly to reduce this kind of changes:

![Uploading image.png…]()

Reply:
@Natim Can we agree on 120?

Reply:
https://github.com/jazzband/django-constance/pull/560/commits/20c9d7765e5c691e744dabccd6161262bcc54634 lets try

Reply:
Just applied formating from scratch to avoid some changes. _forcepushed_

Reply:
Also about the code on your screen. I already have a refactoring of this command in my stash.

Reply:
Yes it looks good to me like that indeed.